go.mod

```diff
@@ -1,9 +1,64 @@
-module github.com/NUS-EVCHARGE/iot-service
+module iot-service/iot_service
```

Comment:
this is changing to use local module. Should we use local or remote? I changing back to remote in this PR as part of the compilation fix. Let me know if has any concern. Open to feedback @gitkh2891 @sweiyang